README.md

```diff
@@ -106,40 +106,18 @@ which is explained below.
 
 ### Disassembly Function
 
-For steps below to work GDB needs to be configured to support pending
-breakpoints. To permanently set this add following to your `.gdbinit`.
+You have to know the function name before hand.
+To disassembly function named `test_function` from binary `test_executable` run
 
 ```
-# Don't ask when restarting the application.
-set confirm off
-# Use pending breakpoints by default.
-set breakpoint pending on
+gdb -ex 'pipe disassemble test_function | tee test_function.asm' -ex 'quit' ./test_executable
 ```
 
-First disassembly function from your binary file using GDB. You have to know the
-function name before hand. To disassembly function named `test_function` from
-binary `test_executable` run
-
-```
-gdb -ex 'b test_function' -ex 'run' -ex 'pipe disassemble | tee test_function.asm' -ex 'quit' ./test_executable
-```
-
-This will set breakpoint at function `test_function`, then
-run the program and disassembly the current function where the breakpoint was
-set to the file `test_function.asm`.
-
 If you don't know the function name you're looking for then you can also list
-all function names using GDB. First set breakpoint at main function and start
-the executable you want with
-
-```
-gdb -ex 'b main' -ex 'run' ./test_executable
-```
-
-Now GDB will break at main function. Now to list all available functions use
+all function names using GDB:
 
 ```
-info functions
+gdb -ex 'info functions' ./test_executable
```

Comment:
When testing this PR I remembered why I added the break point in main, execute and then listing the functions. The reason was because you will also get functions that are dynamically linked to the executable when executed. If only done with command you changed, this will only list functions defined in this executable. That's why I also wanted to use pending break points. I don't think that is possible otherwise.

Original text doesn't make this very clear either and I would like to have that mentioned here. You're welcome to change this text. This can be easily seen with following commands when debugging `ls`
```
gdb -ex 'info functions' ls
gdb -ex 'b main' -ex 'run' -ex 'info functions' ls
```

But I like the idea you have here. Maybe the instructions could be that the break point is in main and then disassembly is made with `pipe disassemble <function_name> | tee <filename>.asm` instead of original command.

Reply:
I see. Can I keep both variants (with and without `b`), with explanation? Breakless method allows printing disassembly for non-native executables (e.g. ARM) via `gdb-multiarch` so it may be important for some users.

Reply:
Good point about disassembling non-native executables. I did't thought of that. I like you method better and easier. Could we add some side note or separate section about dynamic linking when running and using break points? I don't know how useful the breakpoint method will be in general but at least I ended up needing it.

Reply:
Will do.